video_creation/voices.py

```diff
@@ -17,16 +18,34 @@ def save_text_to_mp3(reddit_obj):
     # Create a folder for the mp3 files.
     Path("assets/mp3").mkdir(parents=True, exist_ok=True)
 
-    tts = gTTS(text=reddit_obj["thread_title"], lang="en", slow=False, tld="co.uk")
-    tts.save(f"assets/mp3/title.mp3")
+    if voice == "female":
+        tts = gTTS(text=reddit_obj["thread_title"], lang="en", slow=False, tld="co.uk")
+        tts.save(f"assets/mp3/title.mp3")
+    elif voice == "male":
+        url = 'https://streamlabs.com/polly/speak'
+        body = {'voice': 'Brian', 'text': reddit_obj["thread_title"]}
+        response = requests.post(url, data = body)
+        voice_data = requests.get(response.json()['speak_url'])
+        f = open('assets/mp3/title.mp3', 'wb')
+        f.write(voice_data.content)
+
     length += MP3(f"assets/mp3/title.mp3").info.length
 
     for idx, comment in track(enumerate(reddit_obj["comments"]), "Saving..."):
         # ! Stop creating mp3 files if the length is greater than 50 seconds. This can be longer, but this is just a good starting point
         if length > 50:
             break
-        tts = gTTS(text=comment["comment_body"], lang="en")
-        tts.save(f"assets/mp3/{idx}.mp3")
+
+        if voice == "female":
```

Comment:
Maybe the streamlabs api should be used for both the male and female voices, you just need to find the voice name for the female.

Reply:
we//they use the voices from https://lazypy.ro/tts/ bc it has alot of voices to pick from

Reply:
Yes, female voices are available. I didn't want to remove gTTS though in case people prefer that.